src/dynamodb.ts

```diff
@@ -0,0 +1,93 @@
+import { DynamoDB } from "aws-sdk";
+import { ClientConfiguration, ScanInput } from "aws-sdk/clients/dynamodb";
+import { DateTime } from "luxon";
+import { dynamoLogger } from "./handler";
+// @ts-ignore
+// tslint:disable-next-line:no-var-requires
+const AWSXRay = require("aws-xray-sdk");
```

Comment:
same related issue as discussed previously that without conditional or abstraction will be difficult to test